src/integration/pages/homePage/canonicalTests.js

```diff
@@ -14,12 +14,18 @@ export default ({ service, pageData, displayAds }) => {
     runCanonicalAdsTests();
   }
 
-  it('should render the correct number of curations, including most read & radio schedule', () => {
+  it('should render the correct number of curations, including most read, radio schedule & VJ embed', () => {
     const curationsWithSummaries = pageData.curations.filter(
-      ({ summaries, mostRead, radioSchedule }) =>
-        (summaries && summaries?.length > 0) || mostRead || radioSchedule,
+      ({ summaries, mostRead, radioSchedule, embed }) =>
+        (summaries && summaries?.length > 0) ||
+        mostRead ||
+        radioSchedule ||
+        embed,
     );
-
+    // I have edited the hindi embed html fixture data so that the embed heading is an h2 as this is what we want
+    // Do we need a discussion with VJ to talk about making the headings all h2?
+    // Otherwise if they are going to keep making them h1s, then this test doesn't apply to
+    // all real embeds
```

Comment:
This has now been resolved (see https://test.bbc.com/hindi) - I wonder if we should use the latest version of the embed HTML just in case anything else has been resolved?

Reply:
I'll put the latest in 👍